src/index.css

```diff
@@ -68,7 +68,7 @@ a {
     cursor: pointer;
 }
 
-.table-tracker table {
+.tracker table {
```

Comment:
why do you use this selector?
it will select all tables within .tracker. Is it what you need?

Reply:
Done.Thanks.